flocker/control/_config.py

```diff
@@ -0,0 +1,25 @@
+# XXX move flocker.node._config here, and also add:
+
+
+def marshal_to_application_config_format(deployment):
+    """
+    Convert ``Deployment`` into application configuration format.
+
+    :param Deployment deployment: The current desired configuration.
+
+    :return: Simple Python types suitable for serialization to YAML, in
```

Comment:
I forget if we talked about this already - but maybe JSON instead of YAML?  Perhaps I shouldn't care about this ... but I do :japanese_goblin: :mouse: .